rules/integrations/okta/initial_access_okta_fastpass_phishing.toml

```
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/05/07"
+integration = ["okta"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/05/07"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Detects when Okta FastPass prevents a user from authenticating to a phishing website.
+"""
+
+index = ["filebeat-*", "logs-okta*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Okta FastPass Phishing Detection"
+note = """## Setup
+
+The Okta Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://developer.okta.com/docs/reference/api/system-log/",
+    "https://developer.okta.com/docs/reference/api/event-types/",
+    "https://sec.okta.com/fastpassphishingdetection",
+]
+risk_score = 47
+rule_id = "ee39a9f7-5a79-4b0a-9815-d36b3cf28d3e"
+severity = "medium"
+tags = ["Elastic", "Identity", "Okta", "Continuous Monitoring", "SecOps", "Identity and Access","Initial Access"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:okta.system and event.category:authentication and 
+  okta.event_type:user.authentication.auth_via_mfa and event.outcome:failure and okta.outcome.reason:"FastPass declined phishing attempt"
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1078"
+name = "Valid Accounts"
+reference = "https://attack.mitre.org/techniques/T1078/"
```

```suggestion
id = "T1078"
name = "Valid Accounts"
reference = "https://attack.mitre.org/techniques/T1078/"
```

id = "T1566"
name = "Phishing"
reference = "https://attack.mitre.org/techniques/T1566/"